engine/access/access_test.go

```diff
@@ -399,10 +399,8 @@ func (suite *Suite) TestGetBlockByIDAndHeight() {
 			id := block1.ID()
 			// get block details by ID
 			req := &accessproto.GetBlockByIDRequest{
-				Id: id[:],
-				XFullBlockResponse: &accessproto.GetBlockByIDRequest_FullBlockResponse{
-					FullBlockResponse: true,
-				},
```

Comment:
Can you explain this change?

Reply:
this change is to match the change we made here: https://github.com/onflow/flow/pull/877